addon/classes/Table.js

```diff
@@ -192,6 +192,24 @@ export default class Table extends EmberObject.extend({
   }
 
   /**
+   * The same as `setRows`, however the given array is synced, meaning that
+   * mutating the array also updates the table.
+   *
+   * @method setRows
+   * @param  {Array} rows
+   * @param  {Object} options
+   * @return {Array} rows
+   */
+  setRowsSynced(rows = [], options = {}) {
+    let _rows = RowSyncArrayProxy.create({
+      syncArray: rows,
+      content: emberArray(Table.createRows(rows, options))
```

Comment:
Shoot, I forgot to update the `@method` tag. 

Good question... We could add `options.rowOptions` in the constructor and pass that. If users require options to be passed *right now*, they should be able to wrap in `Table.createRows(rows, options)` methinks.